test/app-tap/datetime.test.lua

```diff
@@ -1667,20 +1667,54 @@ test:test("Time :set{} operations", function(test)
             'hour 6')
     test:is(tostring(ts:set{ min = 12, sec = 23 }), '2020-11-09T04:12:23+0300',
             'min 12, sec 23')
-    test:is(tostring(ts:set{ tzoffset = -8*60 }), '2020-11-08T17:12:23-0800',
+    test:is(tostring(ts:set{ tzoffset = -8*60 }), '2020-11-09T04:12:23-0800',
             'offset -0800' )
-    test:is(tostring(ts:set{ tzoffset = '+0800' }), '2020-11-09T09:12:23+0800',
+    test:is(tostring(ts:set{ tzoffset = '+0800' }), '2020-11-09T04:12:23+0800',
             'offset +0800' )
+    -- timestamp 1630359071.125 is 2021-08-30T21:31:11.125Z
     test:is(tostring(ts:set{ timestamp = 1630359071.125 }),
-            '2021-08-31T05:31:11.125+0800', 'timestamp 1630359071.125' )
-    test:is(tostring(ts:set{ msec = 123}), '2021-08-31T05:31:11.123+0800',
+            '2021-08-30T21:31:11.125+0800', 'timestamp 1630359071.125' )
+    test:is(tostring(ts:set{ msec = 123}), '2021-08-30T21:31:11.123+0800',
             'msec = 123')
-    test:is(tostring(ts:set{ usec = 123}), '2021-08-31T05:31:11.000123+0800',
+    test:is(tostring(ts:set{ usec = 123}), '2021-08-30T21:31:11.000123+0800',
             'usec = 123')
-    test:is(tostring(ts:set{ nsec = 123}), '2021-08-31T05:31:11.000000123+0800',
+    test:is(tostring(ts:set{ nsec = 123}), '2021-08-30T21:31:11.000000123+0800',
             'nsec = 123')
 end)
 
+test:test("Check :set{} and .new{} equal for all attributes", function(test)
```

Comment:
`.new` -> `:new`

Reply:
No, `date.new` could not be an instance method, that's why this syntactic difference to `dateobj:set{}` (which has to be called on object instance).

Reply:
Ah, right.